protocols/v2/const-sv2/src/lib.rs

```diff
@@ -1,114 +1,203 @@
-//! Central repository for all the sv2 constants
+//! This crate serves as the central repository for all constants used in the SV2 protocol.
```

Comment:
again, avoiding words that could be misinterpreted (in this case, "repository")

```suggestion
//! This crate provides all constants used in the SV2 protocol.
```

Reply:
Done here d84c53cc91b97fae08c5b28f90ecd16a7cea9254